src/style.css

```diff
@@ -0,0 +1,214 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+/* styles for header */
+
+header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-link {
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  position: relative;
+  text-decoration: none;
+  padding: 23px 0;
+  margin-right: 20px;
+  color: #000;
+}
+
+.nav-link:last-child {
+  margin: 0;
+}
+
+.nav-link:hover,
+.nav-link:focus {
+  color: #00aedd;
+}
+
+.nav-link:focus::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  left: 0;
+  bottom: 0;
+  background: #00aedd;
+  border-radius: 8px;
+}
+
+.wrap-hed {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 50px 0 40px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px;
+}
+
+.logo > img {
+  width: 100%;
+}
+
+.is-active {
+  color: #00aedd;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  height: 4px;
+  left: 0;
+  bottom: 0;
+  background: #00aedd;
+  border-radius: 8px;
+}
+
+/* styles for product card */
+
+.link {
+  text-decoration: none;

```

Comment:
You duplicated this style.
Maybe better add this to `a`. It will delete underscore from all `a`.